src/riakc_counter.erl

```diff
@@ -31,7 +31,7 @@
 
 -ifdef(EQC).
 -include_lib("eqc/include/eqc.hrl").
--compile(export_all).
+-export([gen_type/0, gen_op/0]).
```

Comment:
heh, sometimes you like export_all, sometimes you don't? No need to change this back, I'm just making a comment.